src/hints/block_context.rs

```diff
@@ -15,19 +15,18 @@ use cairo_vm::types::exec_scope::ExecutionScopes;
 use cairo_vm::types::relocatable::MaybeRelocatable;
 use cairo_vm::vm::errors::hint_errors::HintError;
 use cairo_vm::vm::vm_core::VirtualMachine;
+use indoc::indoc;
 use starknet_api::deprecated_contract_class::ContractClass as DeprecatedContractClass;
 
 use crate::io::classes::write_deprecated_class;
 use crate::io::input::StarknetOsInput;
 use crate::utils::felt_api2vm;
 
-/// Implements hint:
-///
-/// ids.compiled_class_facts = segments.add()
-/// ids.n_compiled_class_facts = len(os_input.compiled_classes)
-/// vm_enter_scope({
-/// 'compiled_class_facts': iter(os_input.compiled_classes.items()),
-/// })
+pub const LOAD_CLASS_FACTS: &str = indoc! {
```

Comment:
Multiline strings with indentation can be used here to get perfectly readable hint code:
```
indoc! {"
    ids.compiled_class_facts = segments.add()
    ids.n_compiled_class_facts = len(os_input.compiled_classes)
    vm_enter_scope({
        'compiled_class_facts': iter(os_input.compiled_classes.items()),
    })"
}
```

Reply:
okay these are not included